### PR TITLE
CV Fer

### PR DESCRIPTION
**Pull request by kimbali**

### CODE REVIEW

File: staff/kim-garcia/cv/index.html [hunk 1]
```diff
@@ -1,94 +1,128 @@
...
+        <hr>
+    </header>
+    <main>
+        <section>
+                <h2>FERNINANDO LEOPOLDO CASAMAYOR</h2>
+                <article>
+                        <figure>
+                                <img src="fer.jpg" alt="foto de leo">
+                        </figure>
+                        <table>
```
Comment (alexgbor) [inline staff/kim-garcia/cv/index.html]: Perhaps you could use ul and li instead of a table.

File: staff/kim-garcia/cv/index.html [hunk 2]
```diff
@@ -1,94 +1,128 @@
...
+                                <tr>
+                                    <td>Nombre comun: Fer para los amigos</td>
+                                </tr>
+                                <tr>
+                                    <td><address>Domicilio: Ciudad de Kiribati</address></td>
+                                </tr>
+                                <tr>
+                                    <td>Fecha de nacimienti: 07/10/1991</td>
```
Comment (alexgbor) [inline staff/kim-garcia/cv/index.html]: typo: nacimienti --> nacimiento

File: staff/kim-garcia/cv/styles.css [hunk 3]
```diff
@@ -1,19 +1,9 @@
 body{
     margin: 0 auto;
     width: 50%;
-    background-color: gold;
+    background-color: aliceblue
```
Comment (alexgbor) [inline staff/kim-garcia/cv/styles.css]: Semicolon.

File: staff/kim-garcia/cv/index.html (hunk 1 above)
Comment (manuelbarzi) [reply staff/kim-garcia/cv/index.html]: agree: for lists use lists, not tables

File: staff/kim-garcia/cv/index.html [hunk 4]
```diff
@@ -1,94 +1,128 @@
...
+        <hgroup>
+                <h1>CURRICULUM VITAE</h1>
+                <h2><cite>“Hay una fuerza motriz más poderosa que el vapor, la electricidad y la energía atómica: la voluntad”</cite></h2>
+        </hgroup>
+        <hr>
+    </header>
+    <main>
+        <section>
+                <h2>FERNINANDO LEOPOLDO CASAMAYOR</h2>
+                <article>
```
Comment (manuelbarzi) [inline staff/kim-garcia/cv/index.html]: why article? take care with its semantics, RTFM => https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article

File: staff/kim-garcia/cv/index.html (hunk 2 above)
Comment (kimbali) [reply staff/kim-garcia/cv/index.html]: done

File: staff/kim-garcia/cv/index.html (hunk 4 above)
Comment (kimbali) [reply staff/kim-garcia/cv/index.html]: done

File: staff/kim-garcia/cv/index.html (hunk 1 above)
Comment (kimbali) [reply staff/kim-garcia/cv/index.html]: done

File: staff/kim-garcia/cv/styles.css (hunk 3 above)
Comment (kimbali) [reply staff/kim-garcia/cv/styles.css]: i removed it

File: staff/kim-garcia/spotify-app/scripts/main.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
+logic.token = "BQCo74c1uYcCS4F90eOmqub9sfqjmffdh4qsQB7nOCUcx8XVGVhvQdiwetXtWyPZ1MgHKlE2ky3K7sFf1C4n6brBLOveBl0kdgTOJQsCzgU_1JBXpWeSqyPEM3kxAqhRN1NNZ1l_4EBRYpiHOvoptCRDuLM"
```
Comment (ghost) [inline staff/kim-garcia/spotify-app/scripts/main.js:1]: you forgot 'use strict'

File: staff/kim-garcia/spotify-app/scripts/main.js [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+    results.forEach(function (result) {
+        var li = document.createElement('li');
+        var a = document.createElement('a');
+
+        a.href = '#/';
+        a.innerHTML = result.name;
+        a.onclick = function () {
+
+            if (this._callback) this._callback(result);
```
Comment (juansensio) [inline staff/kim-garcia/spotify-app/scripts/main.js]: Para mostrar ahora las canciones del álbum deberías definir esta callback (resultsAlbums.onItemClick(...)) y dentro hacer una nueva http request con el id del album para recuperar las canciones y mostrarlas en una nueva lista.

**Review by ghost — CHANGES_REQUESTED** [review]
You should add the tracks list when an album is clicked.

File: staff/kim-garcia/spotify-app/scripts/main.js (hunk 5 above)
Comment (kimbali) [reply staff/kim-garcia/spotify-app/scripts/main.js:1]: done!

File: staff/kim-garcia/arrays-native/scripts/slice.test.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
+
+'use strict';
+
+describe('Array.prototype.slice', function () {
+
+    var array;
+
+    beforeEach(function () {
+
+        array = ["Estoy", "Cortando", "El array", "En dos"];
```
Comment (maiderhernandorena) [inline staff/kim-garcia/arrays-native/scripts/slice.test.js:10]: I like your arrays! : D
Very ingenious haha

File: staff/kim-garcia/spotify-app/scripts/campo-de-tiro.js [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+
+
+        results.updateResults(beer.map(function (result) {
+            return {
+                id: result.id,
+                text: result.name 
+            };
+        }));
+    
+        // detailContainer.clear();
```
Comment (maiderhernandorena) [inline staff/kim-garcia/spotify-app/scripts/campo-de-tiro.js:48]: You should try to delete the code you are not using and spaces between lines. : D

File: staff/kim-garcia/spotify-app/scripts/campo-de-tiro.js [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+    }, this);
+};
+
+ResultsList.prototype.onItemClick = function (callback) {
+    this._callback = callback;
+};
+
+var results = new ResultsList();
+
+results.onItemClick(function (id, text) {
```
Comment (maiderhernandorena) [inline staff/kim-garcia/spotify-app/scripts/campo-de-tiro.js:89]: You should join the parametres to the function: 

`function (id, text) => function(id, text)`

delete the space between the word "function" and the "parentesis"

**Pull request merged**